docker-compose.yml

```diff
@@ -29,9 +31,5 @@ services:
       nofile:
         soft: 1000000
         hard: 1000000
-    deploy:
-      resources:
-        limits:
-          memory: 16384M
```

Comment:
keep this. nethermind still needs it and it wont hurt this

Reply:
Added